src/collective/taxonomy/indexer.py

```diff
@@ -49,8 +49,9 @@ def __call__(self):
                 if identifier in stored_element:
                     found.append((identifier, language, path,))
 
+        lang = getattr(self.context, 'language', language)
```

Comment:
Is it ok to acquire language here? Otherwise probably check with base_hasattr first. See https://github.com/plone/Products.CMFPlone/blob/master/Products/CMFPlone/utils.py#L174